omise/api/resources/customer.py

```diff
@@ -0,0 +1,174 @@
+import copy
+from .base import Base
+from .main_resource import MainResource
+from .lazy_collection import LazyCollection
```

Comment:
for LazyCollection too, I think we should move it to another module not in `api/resources` 😅